paddle/framework/op_registry.h

```diff
@@ -100,13 +100,38 @@ class OpRegistrar : public Registrar {
   }
 };
 
-template <typename PlaceType, typename KernelType>
+template <typename PlaceType, bool at_end, size_t I, typename... KernelType>
+struct OpKernelRegistrarFunctor;
+
+template <typename PlaceType, size_t I, typename... KernelType>
+struct OpKernelRegistrarFunctor<PlaceType, false, I, KernelType...> {
+  using KT = typename std::tuple_element<I, std::tuple<KernelType...>>::type;
```

Comment:
It' hard to associate `KT` with kernel type. I think `KERNEL_TYPE` is better.

Reply:
Done.